src/pages/contributing/get-started.mdx

```diff
@@ -0,0 +1,175 @@
+[app icons](https://www.ibm.com/design/language/iconography/app-icons/production),
+and
+[pictograms](https://www.ibm.com/design/language/iconography/pictograms/contribute)
+on the IBM Design Language site.
+
+## The process
+
+Whether you're contributing code, designs, or documentation, there's a process
+to follow. Here's what it looks like:
+
+### 1. Find a project to work on
+
+#### Volunteer for existing work
+
+The best way to volunteer is to look through
+[existing GitHub issues](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22)
+labeled with `community contribution`. You can also filter by
+[developer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+dev+%F0%9F%A4%96%22_)
+or
+[designer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+design+%3Apencil2%3A%22+)
+roles. Put a comment in the issue saying you'd like the help.
+
+The Carbon team is also happy to help you find a good first issue. Just stop by
+our [office hours](/whats-happening/meetups/) or send us a slack message.
+
+#### Submit an idea
+
+Have a new idea that you think would benefit Carbon? Or do you need to report a
+bug? First, be sure to look through the
+[issue backlog](https://github.com/carbon-design-system/carbon/issues) to make
+sure it is a novel idea or bug. Then, file your proposal on GitHub using the
+[issue templates](https://github.com/carbon-design-system/carbon/issues/new/choose).
+If you're willing to work on this idea yourself, be sure to let is know in your
+issue! Your idea will then go through a triage process by the Carbon team.
+
+### 2. Iterate and get feedback
+
+Most contributors work in groups of 2-3 and either set up weekly sessions or
+join [meetups](/whats-happening/meetups/) such as the Figma Guild, Data Viz
+Guild, or office hours. In these sessions, it is common to share work in
+progress and ask lots of questions. As you make progress, update your GitHub
+issue.
+
+### 3. Review and publish
+
+#### Code deliverables
+
+Work that results in code will be reviewed directly in a pull request.
+Maintainers will be reviewing your work and making comments, asking questions
+and suggesting changes to be made before they merge your code. When you need to
+make a change, commit and push to your branch normally. Once all revisions to
+your pull request are complete, a maintainer will squash and merge your commits
+for you.
+
+#### Contributor License Agreement
+
+The Carbon core team is part of IBM. To accept contributions, we need a signed
+Contributor License Agreement (CLA) from you before code contributions can be
+reviewed and merged. You will be prompted to do this in your PR, but if you have
+questions, please don't hesitate
+to [reach out](help/contact-us/#slack-channels).
```

Comment:
I am getting a 404 for this "reach out" link.

Reply:
thanks! I missed a '/'